sfputil/debug.py

```diff
@@ -0,0 +1,327 @@
+        click.echo(f"{port_name}: Set loopback mode failed. Parameter is not supported")
+        sys.exit(EXIT_FAIL)
+
+    # Output the result of the loopback operation.
+    if status:
+        click.echo(f"{port_name}: {enable} {loopback_mode} loopback")
+    else:
+        click.echo(f"{port_name}: {enable} {loopback_mode} loopback failed")
+        sys.exit(EXIT_FAIL)
+
+
+# Common function to enable/disable TX or RX output
+def set_output(port_name, enable, direction):
+    """
+    Enable or disable TX/RX output based on the direction ('tx' or 'rx').
+
+    This method enables or disables the output for either TX or RX based on the specified direction.
+    It interacts with the SFP object and uses the subport as the channel.
+
+    Args:
+        port_name (str): The port name on which the output is to be enabled/disabled.
+        enable (str): Whether to enable or disable the output. Should be 'enable' or 'disable'.
+        direction (str): The direction of the output. Can be either 'tx' or 'rx'.
+
+    Raises:
+        SystemExit: If there is an error disabling the output or if the direction is not supported.
+    """
+    # Retrieve the SFP object for the given port.
+    sfp = get_sfp_object(port_name)
+    # Retrieve the subport for the given port.
+    subport = get_subport(port_name)
+
+    try:
+        # Enable or disable TX output based on the direction.
+        if direction == "tx":
+            sfp.tx_disable_channel(subport, enable == "disable")
+            click.echo(f"{port_name}: TX output {'disabled' if enable == 'disable' else 'enabled'} on subport {subport}")
+        # Enable or disable RX output based on the direction.
+        elif direction == "rx":
+            sfp.rx_disable_channel(subport, enable == "disable")
+            click.echo(f"{port_name}: RX output {'disabled' if enable == 'disable' else 'enabled'} on subport {subport}")
+    except AttributeError:
+        # If the direction is not applicable for this module, exit.
+        click.echo(f"{port_name}: {direction.upper()} disable is not applicable for this module")
+        sys.exit(ERROR_NOT_IMPLEMENTED)
+    except Exception as e:
+        # Exit if there is an error disabling the output.
+        click.echo(f"{port_name}: {direction.upper()} disable failed due to {str(e)}")
+        sys.exit(EXIT_FAIL)
+
+
+# 'tx-output' subcommand for enabling/disabling TX output
+@debug.command()
+@click.argument('port_name', required=True)
+@click.argument('enable', required=True, type=click.Choice(["enable", "disable"]))
+def tx_output(port_name, enable):
+    """
+    Enable or disable TX output on a port (or specific channel based on subport).
+
+    Args:
+        port_name (str): The port to enable/disable TX output on.
+        enable (str): Whether to enable or disable TX output.
+
+    Raises:
+        SystemExit: If the operation fails for any reason.
+    """
+    # Call the common function to set TX output.
+    set_output(port_name, enable, "tx")
```

Comment:
@vdahiya12 Can we return 0 for success and non-zero for failure so that the user can distinguish between success and failure?

Reply:
I think set_output takes care of exiting with rc if we encounter any errors.